cmd/bundle/init.go

```diff
@@ -40,20 +40,28 @@ func repoName(url string) string {
 func newInitCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "init [TEMPLATE_PATH]",
-		Short: "Initialize Template",
-		Args:  cobra.MaximumNArgs(1),
+		Short: "Initialize a new bundle from a template",
```

Comment:
This is not strictly true though. Mlops stacks for example does not initialize "a bundle".